.github/workflows/deploy_dbt_docs.yaml

```diff
@@ -31,6 +31,9 @@ jobs:
           role-to-assume: ${{ secrets.AWS_IAM_ROLE_TO_ASSUME_ARN }}
           aws-region: us-east-1
 
+      - name: Configure dbt environment
+        uses: ./.github/actions/configure_dbt_environment
```

Comment:
[The error in question](https://github.com/ccao-data/data-architecture/actions/runs/5905883640/job/16020814093):

```
Run dbt docs generate --target "$TARGET"
18:26:[23](https://github.com/ccao-data/data-architecture/actions/runs/5905883640/job/16020814093#step:6:24)  Running with dbt=1.5.5
18:[26](https://github.com/ccao-data/data-architecture/actions/runs/5905883640/job/16020814093#step:6:27):23  Encountered an error:
Runtime Error
  The profile 'athena' does not have a target named ''. The valid target names for this profile are:
   - dev
   - ci
   - prod
Error: Process completed with exit code 2.
```

Previously we had been hardcoding the target `"ci"` so I hadn't noticed that we were missing this step.

Reply:
Nice, I should've caught this too. Tricky tricky